### ISSUE TITLE
Set both side (left and right) for RearViewcontroller

### ISSUE THREAD

**Issue by nhatlee** [John-Lluch/SWRevealViewController#671]
How do I add both: left menu and right menu?

**Comment by iDevelopper** [comment]
Rear and right open in the same time? You can't. Because only the front view controller move, rear and right are static.

**Comment by nhatlee** [comment]
Nope, I mean can I have left menu and right menu (when open left -> right is closed and vice versa
). Like this library:https://github.com/SocialObjects-Software/AMSlideMenu

**Comment by iDevelopper** [comment]
404 for your link

**Comment by iDevelopper** [comment]
Yes, instead of set the action to revealViewController.revealToogle, set it to your own selector in witch you call methods to close left and reveal right for example.

**Comment by nhatlee** [comment]
can you explain more detail, which property I need to set for right menu?

**Comment by nhatlee** [comment]
when user tap right button then right menu will show, and when user tap left button then left menu will show. like this:
https://camo.githubusercontent.com/2f52a3eb68e90024ec5507e7f49b89217a4bd45c/68747470733a2f2f7261772e6769746875622e636f6d2f61727475726465762f414d536c6964654d656e752f6d61737465722f414d536c6964654d656e7544656d6f2d7769746825323053746f7279626f6172642f414d536c6964654d656e752f64656d6f2e676966

**Comment by nhatlee** [comment]
my mainviewcontroller is tabbarviewcontroller, and I need leftmenu and rightmenu is 2 navigationcontroller.

**Comment by iDevelopper** [comment]
Yes then:

```
    @IBOutlet weak var revealButton: UIBarButtonItem!
    @IBOutlet weak var rightButton: UIBarButtonItem!

// ....
        revealButton.target = self.revealViewController()
        revealButton.action = #selector(SWRevealViewController.revealToggle(_:))

        rightButton.target = self.revealViewController()
        rightButton.action = #selector(SWRevealViewController.rightRevealToggle(_:))
```

**Comment by nhatlee** [comment]
Into the example code at appdelegate file:
  ` FrontViewController *frontViewController = [[FrontViewController alloc] init]; -> my Tabbar
    
	RearViewController *rearViewController = [[RearViewController alloc] init];
    
    UINavigationController *frontNavigationController = [[UINavigationController alloc] initWithRootViewController:frontViewController];
    /*-> LeftViewcontroller ->*/ UINavigationController *rearNavigationController = [[UINavigationController alloc] initWithRootViewController:rearViewController]; 

    SWRevealViewController *mainRevealController = [[SWRevealViewController alloc]
        initWithRearViewController:rearNavigationController frontViewController:frontNavigationController];
    
    mainRevealController.delegate = self;
    
	self.viewController = mainRevealController;
	
	self.window.rootViewController = self.viewController;
	[self.window makeKeyAndVisible];`

**where is to set rightviewcontroller?**

**Comment by iDevelopper** [comment]
```
RightViewController` *rightViewController = [[RightViewController alloc] init];
/*-> RightViewcontroller ->*/ UINavigationController *rightNavigationController = [[UINavigationController alloc] initWithRootViewController:rightViewController];
mainRevealController.rightViewController = rightNavigationController;
```
